tests/enip-stats-udp/README.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Test ENIP stats always have `_udp` or `_tcp` prefix
```

Comment:
I don't see how this is tested?

Reply:
As CI failure with current master shows

Reply:
I expect a check for the stats by name, as that is what the work is about I think

Reply:
Right, doing a next iteration.